libs/portals/admin/form-system/src/gql/Form.graphql

```diff
@@ -0,0 +1,606 @@
+mutation formSystemUpdateForm($input: FormSystemUpdateFormInput!) {
+  formSystemUpdateForm(input: $input)
+}
+
+query formSystemGetForm($input: FormSystemGetFormInput!) {
+  formSystemGetForm(input: $input) {
+    form {
+      id
+      name {
+        is
```

Comment:
I realise that this would be a quite big change. But we did a similar thing in the IDS Admin portal were we modelled translated fields. We opted to have the field as an array of `TranslatedValue` type to support possible other language in the future.

You can take a look here and then see references of how we used it. At least this is food for thought.
https://github.com/island-is/island.is/blob/main/libs/api/domains/auth-admin/src/lib/models/translated-value.model.ts